libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -230,7 +230,6 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
         case 'w': {
             printf("Erasing EEPROM...\n");
             AP_Param::erase_all();
-            unlink("dataflash.bin");
```

Comment:
This isn't unused. The erase_all only rewrites the parameter space header and sentinal, it doesn't delete or even shorten the file. This option is supposed to wipe/erase/delete the EEPROM file, not just parameters.

It's actually the erase_all that is extra here.

Reply:
we don't have dataflash.bin... only eeprom. bim

Reply:
You are right, we should correct the name then.

Reply:
Ok I found a way to put it back... the unlink should be done before erase_all !